linkerd/core/src/main/scala/io/buoyant/linkerd/ClearContext.scala

```diff
@@ -0,0 +1,41 @@
+package io.buoyant.linkerd
+
+import com.twitter.finagle._
+import com.twitter.finagle.context.Contexts
+import com.twitter.finagle.tracing.Trace
+
+/** Clears local and broadcast contexts from the request */
+object ClearContext {
+  val role = Stack.Role("ClearContext")
+  val description = "May clear local and broadcast contexts from inbound requests"
+
+  case class Param(enabled: Boolean)
```

Comment:
I guess if there's precedent then I'm okay with it.  It just made me do a double take when reading.